src/data/string/PLString.ts

```diff
@@ -87,4 +88,9 @@ export class PLString
   public count(): PLNumber {
     return plNumber(this.value.length)
   }
+
+  public replace(from: PLString, to: PLString): PLString {
+    const regex = new RegExp(from.value)
+    return new PLString(this.value.replace(regex, to.value))
```

Comment:
I thought you need a simple replace function not one with regexp support.

If you really need the regexp support then we should have 2 variant:
1. with 3 parameters: `from`, `to` and `str`
2. with 4 parameters: `regexp`, `flags`, `to`, and `str` 

For more information see [Flags in regesp](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#flags_in_constructor)